vocode/streaming/agent/prompts/action_prompt.py

```diff
@@ -0,0 +1,21 @@
+ACTION_PROMPT_DEFAULT = """
+You are on a live conversation with a human that is taking place via voice (eg via phone call). You are
+a helpful assistant that will help the human with their needs. In every response, you must do 2 things:
+- respond to the customer with a message
+- decide to take an appropriate action (or none, if it isn't necessary)
+
+The following are the possible actions you can take:
+{actions}
+
+Your responses should look like the following (and should include "Response", "Action", and "Action parameters"):
+
+Response: Yes! Let me look up that information for you.
+Action: look_up_information
+Action parameters: "parameter 1|parameter 2|parameter 3"
```

Comment:
Have you found that GPT will consistently return results formatted like this?

For the dial tone PR, I found that GPT _tended_ to naturally respond with the format `*presses 2*`, so I figured I would explicitly ask for that format in my prompt:
```
...
You will likely need to navigate a phone tree and wait on hold. If you need to push a button on the dial pad, reply with the format '*presses 1*' (or other key) and nothing more.
...
```
But I found that in practice GPT doesn't respond with that format consistently.  I saw situations where it replied `Press 4.` (not "presses", and without the stars).  And I've seen other variations too.

https://github.com/vocodedev/vocode-python/pull/107/files#diff-b086df98bf635ee6a71718985b64311e895fb6a750e1b83f633b6c39baed34c9R65